### PR TITLE
Creates project base and server

### PR DESCRIPTION
**Pull request by virtualdom**
*What:* Creates the base server for the SF movies API.

*Why:* This is a key step in being able to make useful CRUD endpoints for SF movies.

### CODE REVIEW

File: .node-version [hunk 1]
```diff
@@ -0,0 +1 @@
+6
```
Comment (peterdolan) [inline .node-version]: 6.11.1

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+language: node_js
+sudo: false
+node_js:
+  - 6
```
Comment (peterdolan) [inline .travis.yml]: compare against api here

**Review by peterdolan — APPROVED** [review]
LGTM

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+language: node_js
+sudo: false
+node_js:
+  - 6.11.1
+env:
+  - NODE_ENV="test"
+script:
+  - npm test
+  - npm run lint
```
Comment (robinjoseph08) [inline .travis.yml]: change to `yarn`

File: package.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    "lint": "eslint .",
+    "release:patch": "changelog -p && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version patch && git push origin && git push origin --tags",
+    "release:minor": "changelog -m && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version minor && git push origin && git push origin --tags",
+    "release:major": "changelog -M && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version major && git push origin && git push origin --tags",
+    "enforce": "istanbul check-coverage --statement 100 --branch 100 --function 100 --lines 100",
+    "start": "nodemon --ignore test/",
+    "test": "NODE_ENV=test istanbul cover _mocha -- test --require test/setup.js --recursive --timeout 30000",
+    "test:raw": "NODE_ENV=test mocha test --require test/setup.js --recursive --timeout 30000"
+  },
+  "dependencies": {
```
Comment (robinjoseph08) [inline package.json]: super nit: move above `devDependencies`

**Review by robinjoseph08 — APPROVED** [review]
LGTM

**Pull request merged**